src/EntryAPI.php

```diff
@@ -54,4 +54,14 @@ public function getOrganizer($id)
     {
         return $this->api->get('/organizers/' . $id);
     }
+
+    public function createEvent($data): array
+    {
+        return $this->api->post($data, '/events');
+    }
+
+    public function updateEvent($id, $data): array
+    {
+        return $this->api->post($data, '/events/'.$id, "PUT");
+    }
```

Comment:
Can you make sure that the return type array is correct here?

Reply:
Confirmed based on the docs => https://docs.publiq.be/docs/uitdatabank/entry-api/reference/operations/update-a-event#response-body